src/pages/settings/BillingEntity/sections/general/BillingEntityGeneral.tsx

```diff
@@ -65,6 +66,8 @@ const BillingEntityGeneral = () => {
           <>
             <TimezoneBlock billingEntity={billingEntity as BillingEntity} />
 
+            <LocaleBlock />
```

Comment:
If you still want to merge to get the improvements in the translation and in the LanguageSettingsButton, only this line (and the import) needs to be removed and the UI will keep the same.